src/onnx/parse_resize.cpp

```diff
@@ -206,171 +297,202 @@ struct parse_resize : op_parser<parse_resize>
             MIGRAPHX_THROW("PARSE_" + opd.op_name + ": exclude_outside 1 is not supported!");
         }
 
-        // input data shape info
-        auto in_s    = args[0]->get_shape();
-        auto in_lens = in_s.lens();
+        // input data shape info.  Convert static lens to dynamic to simplify referencing them later
+        auto in_s = args[0]->get_shape().to_dynamic();
+        if(in_s.ndim() < 2)
+            MIGRAPHX_THROW(
+                "PARSE_" + opd.op_name +
+                ": requires 2 or more dimensions input, where first dimension is batch #");
```

Comment:
Looks problematic with trying to resize a static shaped tensor without a batch dimension.

Reply:
That's a mistake.  I'll correct the check to only apply with a dynamic input.